easybuild/easyconfigs/s/Stacks/Stacks-2.62-foss-2022a.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'ConfigureMake'
+
+name = 'Stacks'
+version = '2.62'
+
+homepage = 'https://catchenlab.life.illinois.edu/stacks/'
+description = """Stacks is a software pipeline for building loci from short-read sequences, such as those generated on
+ the Illumina platform. Stacks was developed to work with restriction enzyme-based data, such as RAD-seq,
+ for the purpose of building genetic maps and conducting population genomics and phylogeography.
+"""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+source_urls = ['https://catchenlab.life.illinois.edu/stacks/source/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['01c814588fd72c13494785af49123a6a2c20495c924ed84965fdd4e0da1ab524']
```

Comment:
FYI: there was a silent re-release of Stacks v2.62, so checksum needs to be fixed, see #16134

Reply:
Thanks for doing this. These things are a nuisance! I have rebuild and uploaded the test-report for PR #16134